src/main/java/org/gridsuite/explore/server/ExploreException.java

```diff
@@ -14,12 +14,17 @@
 public class ExploreException extends RuntimeException {
 
     public enum Type {
-        STUDY_NOT_FOUND,
-        CONTINGENCY_LIST_NOT_FOUND,
         FILTER_NOT_FOUND,
```

Comment:
why removing STUDY_NOT_FOUND, CONTINGENCY_LIST_NOT_FOUND and keeping FILTER_NOT_FOUND ?
(only rest request error to filter server is checked to generate FILTER_NOT_FOUND exception)

Reply:
well the STUDY_NOT_FOUND, CONTINGENCY_LIST_NOT_FOUND where declared but not used and FILTER_NOT_FOUND was used

Reply:
Yes, but study and contingency could be treated the same way as filters ... (catching exception)

Reply:
yes, but the old behaviour does not define how to treat them,
for me i add an exception handler to treat them globally.
should i keep them like the old behaviour or not ?